charts/rancher-monitoring/v0.2.3/values.yaml

```diff
@@ -142,7 +142,7 @@ exporter-node:
   enabled: false
   apiGroup: "monitoring.coreos.com"
   image:
-    repository: rancher/prom-node-exporter
+    repository: rancher/mirrored-prom-node-exporter
     tag: v1.0.1
   nodeSelectors: []
   resources:
```

Comment:
is wmi_exporter below rancher generated ?

Reply:
Yes, Rancher generates the image https://github.com/rancher/wmi_exporter-package